### DISCUSSION TITLE
Change date format used in console UI

### DISCUSSION THREAD

**Discussion by benr77** [zitadel/zitadel#7537] [Q&A]
In the console UI, dates are shown in US format (MM/DD/YYYY) which is very confusing to non-US users who at least expect DD/MM/YYYY etc. (see `/ui/console/users` created /  modified dates as an example).

Is there any way to change the formatting of dates in the console UI, either on a global or per-user basis?
Thanks

**Comment by hifabienne** [comment]
@peintnermax @eliobischof Is there a possibility for that? Or would it be possible to take the local into account in the console?

**Reply by livio-a** [reply]
From what i see the dates are (mostly) localized based on the user's language. E.g. if I switch to German i get the following:
![image](https://github.com/zitadel/zitadel/assets/9405495/1de46f98-5c1a-40c2-a06b-a1044c7c637f)

I think the problem is, that it's only dependant on the language and would need take the regionality into account as well.
